microsoft/testsuites/core/provisioning.py

```diff
@@ -57,6 +57,7 @@ class Provisioning(TestSuite):
         priority=0,
         requirement=simple_requirement(
             environment_status=EnvironmentStatus.Deployed,
+            network_interface=Synthetic,
```

Comment:
We can add another test case for Synthetic, the smoke test use the default behavior. It means to enable SRIOV, if it supports.

Reply:
thanks, updated